server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java

```diff
@@ -368,27 +368,23 @@ private JettyServerConfig(Map<String, String> configs) {
 
     this.keyStoreType = internalConfig.get(SSL_KEYSTORE_TYPE);
     this.trustStoreType = internalConfig.get(SSL_TRUST_STORE_TYPE);
-    String keyStorePath = null;
-    String keyStorePassword = null;
-    String managerPassword = null;
-    String trustStorePath = null;
-    String trustStorePassword = null;
 
     if (this.enableHttps) {
-      keyStorePath = internalConfig.get(SSL_KEYSTORE_PATH);
-      keyStorePassword = internalConfig.get(SSL_KEYSTORE_PASSWORD);
-      managerPassword = internalConfig.get(SSL_MANAGER_PASSWORD);
-      if (this.enableClientAuth) {
-        trustStorePath = internalConfig.get(SSL_TRUST_STORE_PATH);
-        trustStorePassword = internalConfig.get(SSL_TRUST_STORE_PASSWORD);
-      }
+      this.keyStorePath = internalConfig.get(SSL_KEYSTORE_PATH);
+      this.keyStorePassword = internalConfig.get(SSL_KEYSTORE_PASSWORD);
+      this.managerPassword = internalConfig.get(SSL_MANAGER_PASSWORD);
+    } else {
+      this.keyStorePath = null;
+      this.keyStorePassword = null;
+      this.managerPassword = null;
+    }
+    if (this.enableHttps && this.enableClientAuth) {
+      this.trustStorePath = internalConfig.get(SSL_TRUST_STORE_PATH);
+      this.trustStorePassword = internalConfig.get(SSL_TRUST_STORE_PASSWORD);
+    } else {
+      this.trustStorePath = null;
+      this.trustStorePassword = null;
```

Comment:
I'm not sure why this was changed as it is unrelated to the issues, and I'm not sure this is an improvement.

Reply:
it was changed for #1931 to remove local variables @justinmclean

Reply:
That is fine, but in the future, I would avoid trying to fix multiple different issues in one PR as the PR's description is inaccurate.

Reply:
OK, got it. I will pay attention to it. Thx.